Documentation/ceph-upgrade.md

```diff
@@ -324,109 +340,101 @@ time without compatibility support and without prior notice.
 
 Let's get started!
 
-## 0. Update Ceph to Nautilus version 14.2.5 or higher
-
-As described above in the [Before You Upgrade](#before-you-upgrade) section, you must upgrade to
-Nautilus version 14.2.5 or higher before upgrading to Rook v1.3.
+## 1. Remove Alpha Snapshots
 
-## 1. Migrate Legacy OSDs
-
-As described above in the [Before You Upgrade](#before-you-upgrade) section, it is recommended to migrate legacy OSDs
-before upgrading to Rook v1.3.
+As described above in the [CSI Snapshots](#csi-snapshots) section, if you are using alpha snapshots
+it is required to remove them before the upgrade since they are not compatible with beta snaphots
+available with Ceph-CSI v3.0.
 
 ## 2. Update the RBAC and CRDs
 
 > Automatically updated if you are upgrading via the helm chart
 
-First apply new resources. This includes slightly modified privileges (RBAC) needed by the Operator.
-Also update Ceph Custom Resource Definitions (CRDs) at this time. Many CRDs have had a `status` item
-added to them which must be present for Rook v1.3 to update CRD statuses.
+First apply new resources. This includes slightly modified privileges (RBAC) needed by the Operator
+and updates to the Custom Resource Definitions (CRDs).
 
 ```sh
-kubectl apply -f upgrade-from-v1.2-apply.yaml -f upgrade-from-v1.2-crds.yaml
+kubectl delete -f upgrade-from-v1.3-delete.yaml
+kubectl apply -f upgrade-from-v1.3-apply.yaml -f upgrade-from-v1.3-crds.yaml
 ```
 
-## 3. Update Ceph CSI version to v2.0
+## 3. Update Ceph CSI version to v3.0
 
-If you have specified custom CSI images in the Rook-Ceph Operator deployment, you should update your
-the deployment to use the latest Ceph-CSI v2.0 and related drivers. If you have not specified custom
-CSI images in the Operator deployment this step is unnecessary since Rook v1.3 will use the latest
-drivers by default.
+> Rook v1.4 will install Ceph-CSI v3.0 use the latest drivers by default.
+> If you have not specified custom CSI images in the Operator deployment this step is unnecessary.
 
-See the section [CSI Updates](#csi-updates) for details about how to do this.
+If you have specified custom CSI images in the Rook-Ceph Operator deployment, it is recommended to update to use the latest Ceph-CSI v3.0 driver.
+See the section [CSI Version](#csi-version) for more details.
 
 ## 4. Update the Rook Operator
 
 > Automatically updated if you are upgrading via the helm chart
 
-The largest portion of the upgrade is triggered when the operator's image is updated to `v1.3.x`.
+The largest portion of the upgrade is triggered when the operator's image is updated to `v1.4.x`.
 When the operator is updated, it will proceed to update all of the Ceph daemons.
 
 ```sh
-kubectl -n $ROOK_SYSTEM_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.3.0
+kubectl -n $ROOK_SYSTEM_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.4.0
```

Comment:
Should we go with  `ROOK_NAMESPACE` instead? Or is it for legacy support?

Reply:
Yes that's for legacy support. This var would be the same as ROOK_NAMESPACE if it's a more recent cluster.